content/en/docs/reference/command-line-tools-reference/feature-gates/selinux-mount.md

```diff
@@ -16,5 +16,5 @@ recursively.
 It widens the performance improvements behind the `SELinuxMountReadWriteOncePod`
 feature gate by extending the implementation to all volumes.
 
-Enabling the `SELinuxMount` feature gate requires the feature gate `SELinuxMountReadWriteOncePod` to
-be enabled.
+Enabling the `SELinuxMount` feature gate requires the feature gates `SELinuxMountReadWriteOncePod`
+and `SELinuxChangePolicy` to be enabled.
```

Comment:
Just out of curiosity ...
this sounds like a stricter requirement than before, so it is not backward compatible ..
so ...
Please help double check if that is the case.

Reply:
It is stricter, however, `SELinuxMount` was alpha and I think such changes are allowed.
Does it need a stronger warning? Do we actually support upgrade with alpha features?

Reply:
Given that `SELinuxMount` was Alpha, this should be okay. Thanks.